### DISCUSSION TITLE
High resolution ocean model with animated camera

### DISCUSSION THREAD

**Discussion by jbusecke** [pyvista/pyvista#4937] [Show and tell]
I was [encouraged](https://github.com/bjlittle/geovista/issues/15#issuecomment-1720205373) to post this video here: 

<img width="990" alt="image" src="https://github.com/pyvista/pyvista/assets/14314623/654ac05e-d6c7-44ed-945c-dad84903b487">

https://vimeo.com/863996198?share=copy

It visualizes surface output of a high resolution coupled climate model (CM2.6) run at [NOAA-GFDL](https://www.gfdl.noaa.gov/high-resolution-climate-modeling/). I did this visualization for the [M²LInES](https://m2lines.github.io) project.

The fields shown are the surface velocity magnitude which is transitioning to surface salinity.

**Comment by bjlittle** [comment]
@jbusecke Phoar! 🤩

**Comment by MatthewFlamm** [comment]
Wow!  Great visualization.

**Comment by tkoyama010** [comment]
@jbusecke Thanks for sharing great work!
